### PR TITLE
Pin AzureCore version

### PR DESCRIPTION
**Pull request by tjprescott**
Avoid breaking Autorest due to development changes to AzureCore.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ let package = Package(
         .package(url: "https://github.com/tjprescott/Yams.git", .branch("CircularReferences")),
         .package(url: "https://github.com/stencilproject/Stencil.git", .branch("trim_whitespace")),
         .package(url: "https://github.com/apple/swift-nio", from: "2.0.0"),
-        .package(name: "AzureSDK", url: "https://github.com/Azure/azure-sdk-for-ios.git", .branch("master")),
+        .package(name: "AzureSDK", url: "https://github.com/Azure/azure-sdk-for-ios.git", from: "1.0.0-beta.2"),
```
Comment (sacheu) [inline Package.swift:16]: i think we need to make the same change in Package.stencil.

**Review by sacheu — CHANGES_REQUESTED** [review]
we need to change Package.stencil file too.

**Comment by tjprescott** [comment]
@sacheu added the line to package.stencil

**Comment by sacheu** [comment]
> @sacheu added the line to package.stencil

Thanks Travis. I think all the Package.swift in test/integerated/generated should be re-generated.

**Review by sacheu — APPROVED** [review]

**Comment by tjprescott** [comment]
This seems to be fixed. Merging.

**Pull request merged**